Sources/Public/Animation/LottieAnimationLayer.swift

```diff
@@ -703,9 +706,19 @@ public class LottieAnimationLayer: CALayer {
     }
   }
 
-  public var animationView: LottieAnimationView? {
-    set { rootAnimationLayer?.animationView = newValue }
-    get { rootAnimationLayer?.animationView }
+  /// Whether or not the Main Thread rendering engine should use `forceDisplayUpdate()`
+  /// when rendering each individual frame.
+  ///  - The main thread rendering engine implements optimizations to decrease the amount
+  ///    of properties that have to be re-rendered on each frame. There are some cases
+  ///    where this can result in bugs / incorrect behavior, so we allow it to be disabled.
+  ///  - Forcing a full render on every frame will decrease performance, and is not recommended
+  ///    except as a workaround to a bug in the main thread rendering engine.
+  ///  - Has no effect when using the Core Animation rendering engine.
+  public var mainThreadRenderingEngineShouldForceDisplayUpdateOnEachFrame = false {
+    didSet {
+      (rootAnimationLayer as? MainThreadAnimationLayer)?.forceDisplayUpdateOnEachFrame
+        = mainThreadRenderingEngineShouldForceDisplayUpdateOnEachFrame
```

Comment:
```suggestion
        = newValue
```

Reply:
`didSet` doesn't provide a `newValue` property, only `set` and `willSet` provide that